tests/adapter/mongo/test_query.py

```diff
@@ -48,10 +48,10 @@ def test_panel_query(real_populated_database, case_obj, variant_objs):
         genes = [ 17284 ] # POT1 gene
     )
     adapter.load_hpo_term(hpo_term)
-    assert adapter.hpo_term_collection.find().count() == 1
+    assert sum(1 for i in adapter.hpo_term_collection.find()) == 1
 
     # no variants in database
-    assert adapter.variant_collection.find().count() == 0
+    assert sum(1 for i in adapter.variant_collection.find()) == 0
```

Comment:
I see you have a huge amount of sums in the tests, so it's maybe better this way than the find_one?